components/ArticleCard/ArticleDropDown.tsx

```diff
@@ -0,0 +1,23 @@
+import ArticleImg from './ArticleImg';
+import DropDown from '../DropDown';
+
+function ArticleDropDown({ image }: { image: string | null }) {
+  return (
+    <div className="ml-auto flex">
+      {image !== null && <ArticleImg src="/images/codeitprofile.png" />}
+
+      <div className="mo:absolute mo:bottom-pr-29 mo:right-pr-16 mo:ml-0">
+        <DropDown
+          trigger={<button className="icon-kebab ml-pr-16" />}
+          items={[
```

Comment:
margin-left를 굳이 button에 직접 주지 않아도 될것같습니다!

```suggestion
      <div className="ml-pr-16 mo:absolute mo:bottom-pr-29 mo:right-pr-16 mo:ml-0">
        <DropDown
          trigger={<button className="icon-kebab" />}
```

Reply:
아 이해했습니다! 수정하겠습니다!